themes/digital.gov/layouts/communities/list.html

```diff
@@ -40,24 +40,16 @@ <h1>{{- .Title | markdownify -}} </h1>
             {{ range $i, $e := $dg_communities }}
 
             <li class="usa-card tablet:grid-col-4">
-              <div class="usa-card__container">
+							<div class="usa-card__container">
+								<div class="usa-card__cop-bar"></div>
                 <div class="usa-card__header">
                   <h2 class="usa-card__heading">{{- $e.Title | markdownify -}}</h2>
                 </div>
-                <div class="usa-card__media">
-                  <div class="usa-card__img">
-                    <img
-                      src="{{- $e.Params.dg_image | absURL -}}"
-                      alt=""
-                      aria-hidden="true"
-                    />
-                  </div>
-                </div>
                 <div class="usa-card__body">
                   <p>{{- $e.Params.summary | markdownify -}}</p>
                 </div>
                 <div class="usa-card__footer">
-                  <a class="usa-button" href="{{- $e.Permalink -}}" title="{{- $e.Title -}}">
+									<a class="usa-button" href="{{- $e.Permalink -}}" title="{{- $e.Title -}}">
```

Comment:
Can we add some hover/active/focus states to these buttons?

Reply:
Reverted back to existing button styles